### ISSUE TITLE
CupertinoNavigationBar fully transparent state

### ISSUE THREAD

**Issue by ton252** [flutter/flutter#85443]
There is no way to put a completely transparent background, in iOS such an opportunity is present.
The problem is in **nav_bar.dart line 122**
```
 if (backgroundColor.alpha == 0xFF)
```
Full description of the problem:
https://stackoverflow.com/questions/55240765/fully-transparent-cupertinonavigationbar-possible

It might be worth adding a flag like **fullyTransparent** and do something like that:
```
 if (backgroundColor.alpha == 0xFF || fullyTransparent)
```
or move this validation in public widget function that it can be override later

**Comment by darshankawar** [comment]
Similar / related : https://github.com/flutter/flutter/issues/60411

**Comment by KirXray** [comment]
Hello everyone, is this problem still not fixed?

**Comment by kerberjg** [comment]
Is this a confirmed duplicate of #60411? If so it's been resolved via #149102

**Comment by victorsanni** [comment]
Hi @ton252, it seems like this has been resolved by https://github.com/flutter/flutter/pull/149102. Feel free to reopen the issue if not.
